doc/ref_model/chapters/chapter01.md

```diff
@@ -38,18 +38,18 @@ This section introduces the high-level principles of infrastructure abstraction
    - Networking resources (Limited to connectivity services only)
    - Acceleration resources
 1. Cloud Infrastructure exposed resources should be supplier independent
-1. All Cloud Infrastructure Application Programming Interfaces (APIs) must ensure Interoperability (multi-vendor, components substitution), drive Simplification, and open source implementations that have an open governance model (e.g. come from Open Communities or Standards Development Organisations). Through such APIs will cloud infrastructure resources be discovered/monitored by management entities, configured on behalf of VNFs and consumed by VNFs.
-1. VNFs should be modular and be designed to utilise the minimum resources required for the service
+1. All Cloud Infrastructure Application Programming Interfaces (APIs) must ensure Interoperability (multi-vendor, components substitution), drive Simplification, and open source implementations that have an open governance model (e.g. come from Open Communities or Standards Development Organisations). Through such APIs will cloud infrastructure resources be discovered/monitored by management entities, configured on behalf of workloads and consumed by workloads.
```

Comment:
API is Application Program Interface 

"...open source implementations that have an open governance model..." -- I think the intent here is to say that the "open source implementations" should be from sources with an "open governance model" but that is not what it says.

Reply:
This could be the subject of some debate, and if this is genuinely of concern then please create an issue to track this.  This PR is focused on changing wording and so we should allow it to remain narrowly focused.

Reply:
@kedmison The current wording has issues and can be handled either by refining or deleting some words.

Reply:
@pgoyal01 @kedmison as we do not have a consensus about the topic here I would suggest to create a separate ticket to address it.

Reply:
Issue #1542 opened